tests/empty-state.test.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+function set_up_before_script() {
+    source bin/git-forgit
+}
+
+function set_up() {
+    # Create a temporary git repository for testing
+    TEST_DIR=$(mktemp -d)
+    cd "$TEST_DIR" || exit 1
```

Comment:
Could we not use `temp_dir` which gets cleaned up automatically? This way we also wouldn't need the `tear_down` function.

```suggestion
    cd "$(temp_dir)" || return 1
```